app/Http/Controllers/ArticleCategoryController.php

```diff
@@ -2,8 +2,13 @@
 
 namespace App\Http\Controllers;
 
-use App\Models\ArticleCategory;
+use App\Models\Article;
 use Illuminate\Http\Request;
+use App\Models\ArticleCategory;
+use Dflydev\DotAccessData\Data;
+use Illuminate\Support\Facades\Session;
+use App\Http\Requests\CategoryArticleRequest;
+use App\Http\Requests\updateCategoryArticleRequest;
```

Comment:
Personally, I don't like to add uses without the need to it 
```php
use App\Models\Article;
use Illuminate\Http\Request;
use Dflydev\DotAccessData\Data;
use Illuminate\Support\Facades\Session;
```
These are never used in your controller